tests/test_config_node.py

```diff
@@ -67,6 +68,17 @@ def test_get_rose_vars_from_config_node__unbound_env_var(caplog):
     assert exc.match('env=X: MYVAR: unbound variable')
 
 
+def test_get_rose_vars_from_config_node__ignores_ROSE_VERSION(caplog):
+    """It should fail if variable unset in environment.
+    """
+    ret = {}
+    node = ConfigNode()
+    node.set(['template variables', 'ROSE_VERSION'], 99)
+    get_rose_vars_from_config_node(ret, node, {})
```

Comment:
Can we add a test to ensure the `ROSE_VERSION` is being ovewritten correctly. Oh and can we test Cylc too.

Reply:
Always up for more testing.

Reply:
Hopefully https://github.com/cylc/cylc-rose/pull/86/commits/150af3743c4b6da83ab6fafd67d971a3ad3bf666 answers...